.circleci/config.yml

```diff
@@ -1980,7 +1980,7 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - X-dont-run
```

Comment:
Instead can we delete it from the file (as we did for other AWS workflows) since we won't be using aws summary anymore.

Reply:
Ok, done with that.